src/python/deepgnn/graph_engine/snark/BUILD

```diff
@@ -16,7 +16,8 @@ py_library(
         "convert.py",
         "meta_merger.py",
         "converter/__init__.py",
-        "converter/json_converter.py",
+        "converter/process.py",
```

Comment:
this is a breaking change, could you add a note in the changelog.md?

Reply:
The breaking change is the converter argument right? Users should never import the json_converter since it only has writers.

Reply:
no, you removed one public function.

> Users should never import the json_converter since it only has writers.

That's not how conventions work: https://peps.python.org/pep-0008/#method-names-and-instance-variables